src/main/groovy/nebula/plugin/publishing/ivy/PlatformDependencyVerifier.groovy

```diff
@@ -52,7 +52,6 @@ class PlatformDependencyVerifier {
         Map<String, Set<? extends ModuleIdentifier>> dependencyMap = [:]
         dependencyMap['runtime'] = platformDependencies(configurations.runtimeClasspath)
         dependencyMap['compile'] = platformDependencies(configurations.compileClasspath)
-        dependencyMap['compileOnly'] = platformDependencies(configurations.compileOnly)
```

Comment:
We don't see people putting recommendation into `compileOnly` right?

Reply:
not really, it is weird.